### ISSUE TITLE
Require Laravel's Arr helper, to allow ->only filtering on Requests get/post inputs

### ISSUE THREAD

**Issue by AnnoyingTechnology** [polyfony-inc/polyfony#258]
![Screenshot from 2019-08-05 10-44-07](https://user-images.githubusercontent.com/5336824/62451196-11893100-b76e-11e9-8fef-84f91f12d59e.png)

We could then do 

```php

class DemoController extends Controller {

	public function updateAction() {

		// we could now accept an array
		$user_updates = Request::post([
		   'firstname',
		   'lastname',
		   'phone',
		   'address',
		   'city',
		   'zip_code',
		   'id_country'
		])

		// and retrieve only those, allowing for 
		(new User)
		   ->set($user_updates)
		   ->save();

	}

}

```

currently we, if we can't trust the whole post, we have to do this
```php

// update some fields
$user->set([
	'login'					=>Request::post('Accounts')['login'],
	'id_level'					=>Request::post('Accounts')['id_level'],
	'is_enabled'				=>Request::post('Accounts')['is_enabled'],
	'account_expiration_date'	=>Request::post('Accounts')['account_expiration_date']
]);
```
which is lengthy, and can produce notices/warning when an array key is not defined/posted.

poke @WarenN

**Comment by AnnoyingTechnology** [comment]
Possible syntax :


```php

Request::get('Sites', [
    '',
    '',
    '',
]);


```

**Comment by AnnoyingTechnology** [comment]
fixed in #e0b58de
